### DISCUSSION TITLE
Bird brain move - Deleted PW Container by mistake

### DISCUSSION THREAD

**Discussion by rwskinner** [jasonacox/Powerwall-Dashboard#507] [Q&A] [answered]
First, thanks for such a great program!  Love it.

Now, for the bad part. I deleted the Powerwall wall container by mistake on my Windows 11 Docker Desktop. Powerwall 3
I ran the Powerwall-Dashboard setup again, and all my info was still in there.  I was able to go the the PW Dashboard okay and everything seems to work, but the Tesla Battery Power Flow animation is a static drawing, no info and nothing happening.

All the images in the container are running so I looked at the powerwall logs.

Any help would be greatly appreciated.

07/28/2024 03:39:56 PM [proxy] [INFO] pyPowerwall [0.10.8] Proxy Server [t63] - HTTP Port 8675
07/28/2024 03:39:56 PM [proxy] [INFO] pyPowerwall Proxy Started
07/28/2024 03:39:57 PM [proxy] [INFO] pyPowerwall Proxy Server - Cloud Mode
07/28/2024 03:39:57 PM [proxy] [INFO] Connected to Site ID 2252339090761259 (My Home)
07/28/2024 03:40:02 PM [pypowerwall.cloud.pypowerwall_cloud] [WARNING] This API [get_api_auth_toggle_supported] is using mock data in cloud mode. This message will be printed only once at the warning level.
07/28/2024 03:40:47 PM [pypowerwall.cloud.pypowerwall_cloud] [WARNING] This API [get_api_sitemaster] is using mock data in cloud mode. This message will be printed only once at the warning level.
07/28/2024 03:40:47 PM [pypowerwall.cloud.pypowerwall_cloud] [WARNING] This API [get_api_powerwalls] is using mock data in cloud mode. This message will be printed only once at the warning level.

**Comment by rwskinner** [comment]
I forgot to add - The energy usage chart is showing the battery level and shows it slow dropping during my peak hours, and solar comes from the power wall CT and it appears to be reading properly.  So everything really seems okay but the animated graphic.

**Reply by rwskinner** [reply]
Solved - Everything worked fine in Incognito mode so I cleared my cache, logged back in and it all worked fine.

**Reply by jasonacox** [reply, marked as answer]
Glad you got it working! 

In case anyone else ever accidentally deletes the pypowerwall container, instead of runing `setup.sh` you can just run this from the Powerwall-Dashboard folder:

```bash
./compose-dash.sh up -d
```

That causes docker compose to ensure everything is set up and running as defined by your config.

I'm surprised a simple refresh didn't reload the power flow animation on your dashboard. Is it possible you had more than one dashboard tab open? You can get a "blank graphic" in the Chrome browser if you have the animation open in another tab/window (for some reason Chrome won't let that iframe load more than once). Safari, Brave and Firefox seems to allow it, but not Chrome. 🤷

**Reply by rwskinner** [reply]
That's good to know on using the compose script.  Yes,  I did have two windows open and didn't realize it.
